frontend/express/public/javascripts/countly/vue/components/helpers.js

```diff
@@ -140,4 +140,21 @@
                     </div>'
     }));
 
+    Vue.component("cly-tooltip-icon", countlyBaseComponent.extend({
+        mixins: [
+            _mixins.i18n
```

Comment:
Not sure if we still want to add the Int mixins even though we are not using them inside the component (just in case you missed it, otherwise I don't think it has a major impact).

Reply:
makes sense, we can remove it if its not being used.